mw-webapp/src/component/header/Header.tsx

```diff
@@ -49,7 +49,7 @@ const renderNavigationLinks = (navigationLinks: (navigationLink | null)[]) => {
  * Header component
  */
 export const Header = () => {
-  const [user, setUser] = useState<User | null>(null);
+  const {user, setUser} = useUserContext();
```

Comment:
why does header responsible for handle auth state? why not the app?

Reply:
deleted setUser from Context